event_attribution_reporting.md

```diff
@@ -50,7 +50,7 @@ dictionary AttributionSourceParams {
 }
 ```
 
-When invoked the browser will directly add the specified source to storage, and return whether the browser was successful in parsing the params.
+When invoked the browser will directly add the specified source to storage, and resolve or reject the promise based on whether the browser was successful in parsing the params and storing the source.
```

Comment:
We should think about whether failure to store the source leaks unintended information back to the caller.

Reply:
We should mention in the explainer that, due to privacy reasons, some failure states (those that depend on cross site information) may not show up as rejections.

Reply:
Done.